### ISSUE TITLE
Page refreshes during disposition user interaction

### ISSUE THREAD

**Issue by yoomlam** [department-of-veterans-affairs/caseflow#12461]
### User or job story
User story: As a Judge with a long list of hearings in my Daily Docket, I need to set a timed hold for a case with minimal cognitive disruption, so that I'm able to focus on a single case.

### Acceptance criteria
- Page does not refresh (and hence scrolls to the top of the page) after setting the disposition, particularly for a case that is listed lower on the page.

### Release notes
Allows a judge to set a timed hold or disposition without page refresh

#### Designs
After setting the Disposition to 'Held', the page refreshes and shows the green success banner. The judge must then scroll down, searching for the case that was just modified, to set the Hold Open field.
![image](https://user-images.githubusercontent.com/55255674/67398250-5b690280-f578-11e9-9336-ee3aa3a2b0c6.png)

The workaround is to set the Hold Open field first (which doesn't refresh the page), then set the Disposition.

#### Background/context
Value statement - Reduces cognitive disruption so that judge is able to focus on a single case at a time.

#### Success criteria
Judge is able to focus on a single case at a time without scrolling to search for current case.

#### Technical notes
Find in Caseflow Hearings, Daily Docket.

**Comment by cabeaulac** [comment]
Yeah, page refresh outside any user interaction with the page is a bad experience. Independent scrolling of a page while a user is viewing it also degrades the experience.
